README.md

```diff
@@ -419,7 +419,43 @@ On Linux, it is often necessary to first update `pip` to a version that supports
 
 ### pip installation: GPU (CUDA)
 
-If you want to install JAX with both CPU and NVidia GPU support, you must first
+JAX provides pre-built CUDA wheels for **Linux only**. Other
+operating systems are possible, but require [building from
+source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source).
+
+There are two ways to install JAX with CUDA support, either by installing CUDA
+using `pip` or by using a self-installed CUDA.
+
+#### Using a `pip`-installed copy of CUDA and CuDNN (recommended)
+
+NVIDIA provides almost everything JAX needs as wheels on PyPi with the exception
+of the GPU drivers. Installing CUDA from PyPi is the easiest way to install JAX
+and we strongly recommend it. The main downside is that NVIDIA's CUDA wheels are
+quite large (approximately 2GB).
```

Comment:
This 2GB size seems concerning to me if we are going to tell people to use this as the default path. 

Downloading such large files everywhere is not possible.

Reply:
Note you're going to download the same things either way. It's just whether you download it via pip or some other way (apt, for example). I want to strongly encourage users to use this path to lower our support burden.

Added some text.

Reply:
Ohh I see. That was not clear!

LGTM